src/database/models.py

```diff
@@ -571,14 +575,24 @@ class Team(models.Model):
     about this goal
   """
   id = models.AutoField(primary_key=True)
-  #Team names should be unique globally
+  #Team names should be unique globally (Not!)
   name = models.CharField(max_length=SHORT_STR_LEN)
+  tagline = models.CharField(max_length=SHORT_STR_LEN, blank=True)
   description = models.TextField(max_length=LONG_STR_LEN, blank=True)
+
   admins = models.ManyToManyField(UserProfile, related_name='team_admins', 
     blank=True) 
   members = models.ManyToManyField(UserProfile, related_name='team_members', 
     blank=True) 
+
+  # may change this from ForeignKey to ManyToManyField to allow team to span communities
   community = models.ForeignKey(Community, on_delete=models.CASCADE)
+  image = models.ManyToManyField(Media, related_name='team_image')                         # 0 or more photos - could be a slide show
+  video = models.ForeignKey(Media, on_delete=models.SET_NULL, related_name='team_video', null=True)                               # allow one video
```

Comment:
i dont know but it think its always a better idea for people to upload videos on youtube or vimeo and then insert a link.  I strongly advise against storing "video" files ourselves.

Reply:
Good point.  Changed to video_link